packages/studiocms_devapps/src/utils/wp-api/index.ts

```diff
@@ -110,9 +114,60 @@ export const importPostsFromWPAPI = async (endpoint: string, useBlogPkg: boolean
 	try {
 		for (const post of posts) {
 			console.log('importing post: ', post.title.rendered);
-			await importPost(post, useBlogPkg);
+			await importPost(post, useBlogPkg, endpoint);
 		}
 	} catch (error) {
 		console.error('Failed to import posts from WP-API: ', error);
 	}
 };
+
+export const importSettingsFromWPAPI = async (endpoint: string) => {
+	const url = apiEndpoint(endpoint, 'settings');
+
+	console.log('Fetching site settings from: ', url.origin);
+
+	const response = await fetch(url);
+	const settings: SiteSettings = await response.json();
+
+	console.log('Importing site settings: ', settings);
+
+	let siteIcon: string | undefined = undefined;
+
+	if (settings.site_icon_url) {
+		siteIcon = await downloadPostImage(settings.site_icon_url, ASTROPUBLICFOLDER);
+	}
+
+	if (!settings.site_icon_url && settings.site_logo) {
+		const siteLogoURL = apiEndpoint(endpoint, 'media', `${settings.site_logo}`);
+		const siteLogoResponse = await fetch(siteLogoURL);
+		const siteLogoJson = await siteLogoResponse.json();
+		siteIcon = await downloadPostImage(siteLogoJson.source_url, ASTROPUBLICFOLDER);
+	}
+
+	const siteConfig: typeof tsSiteConfig.$inferInsert = {
+		id: 1,
```

Comment:
Will this always be `1`?

Reply:
This is OUR config so yes (There is a variable for it but it was easier to just not import it from core... :sweat_smile: )

Reply:
done [946a670](https://github.com/astrolicious/studiocms/pull/367/commits/946a6704f5757b4b92cebd235a2616adbd18e9cf)